### ISSUE TITLE
mean loss inf - batch loss: inf

### ISSUE THREAD

**Issue by debasish-mihup** [burchim/EfficientConformer#5]
During the training, I have around 9000+ batches of each having 32 audio segment data. After running the training for 1 epoch the mean loss and batch loss haven't changed from inf. Can you give some idea where the problem might be?

**Comment by debasish-mihup** [comment]
Looks like the downsampling and nature of the data did not allow CTC model to train properly. I trained the Efficient Conformer model which worked OK. If the author has any suggestions, I can try them. Closing the issue.

**Comment by burchim** [comment]
Hi,

The format of your text data does not seem appropriate. The transcriptions of your audio samples should be formatted into 'plain text'. Otherwise, the byte-pair-encoding tokenizer may not be trained correctly. This will also be easier to understand the output of the model.

**Comment by debasish-mihup** [comment]
> Hi,
> 
> The format of your text data does not seem appropriate. The transcriptions of your audio samples should be formatted into 'plain text'. Otherwise, the byte-pair-encoding tokenizer may not be trained correctly. This will also be easier to understand the output of the model.

Sorry I did not reply to this comment earlier. The reason you are seeing the transcript data in that form is that it is phoneme data of the transcript (phonemes have been encoded to single character symbols). The reason for this is too handle segments containing multiple language input. I did run the tokenizer on this encoded character script but was still unable to run the CTC model to converge. I was able to train efficient transducer model.
